<launch>
   
   <include file="$(find turtlebot3_gazebo)/launch/turtlebot3_empty_world.launch" />
   <!--<include file="$(find turtlebot3_teleop)/launch/turtlebot3_teleop_key.launch" />-->
   
   <node name="turtlebot_mpc_node" pkg="turtlebot_mpc" type="turtlebot_mpc_node" output="screen" >
      <param name="mpc_config_path" value="$(find turtlebot_mpc)/cfg/mpc_config.yaml" />
      <param name="xref_perp_path" value="$(find turtlebot_mpc)/cfg/state.csv" />
      <param name="uref_perp_path" value="$(find turtlebot_mpc)/cfg/input.csv" />
      <param name="xref_par_path" value="$(find turtlebot_mpc)/cfg/state3.csv" />
      <param name="uref_par_path" value="$(find turtlebot_mpc)/cfg/input3.csv" />

      <!--<rosparam command="load" file="$(find turtlebot_mpc)/cfg/mpc_config.yaml" />-->
   </node>

</launch>
